aiosmtpd/docs/NEWS.rst

```diff
@@ -2,6 +2,16 @@
  NEWS for aiosmtpd
 ===================
 
+1.2.3a3
+=======
+
+* Convert all test cases to pytest (Closes #198)
+* Lots of new test cases
+* Updated documentation
+* Travis now tests on Python 3.9 as well
+* Travis testing is now simplified and much faster
+
+
```

Comment:
Thinking of removing mentions of Travis as we're moving over to ... something other than Travis (GitHub Actions?)